Dockerfile

```diff
@@ -69,11 +69,17 @@ ENV PYTHONPATH="/opt:$PYTHONPATH"
 COPY --from=builder /opt/venv /opt/venv
 ENV PATH="/opt/venv/bin:$PATH"
 
-# START DEBUGGING
```

Comment:
@posterzh do you think these lines are still helpful? I haven't used them in a long time so maybe they are just noise?

Reply:
could always remove and put in documentation or something like that perhaps?